addons/mail/static/src/core/web_portal/message_patch.js

```diff
@@ -136,8 +136,8 @@ patch(Message.prototype, {
             iconellipsisEl.className = "fa fa-ellipsis-h fa-lg";
             ellipsisbtnEl.append(iconellipsisEl);
             group[0].parentNode.insertBefore(ellipsisbtnEl, group[0]);
-            // Toggle All next nodes
-            if (!this.state.isReadMoreByIndex.has(index)) {
+            // Toggle all nodes except reply nodes
+            if (!group[0].querySelector('.o_mail_reply_content')) {
```

Comment:
```suggestion
            // Toggle all nodes except reply nodes
            if (
                 !this.state.isReadMoreByIndex.has(index) &&
                 !group[0].querySelector('.o_mail_reply_content')
               )
```

You could update it this way. This keeps the original check to ensure the index isn't already set, while also skipping groups that contain reply nodes